### PR TITLE
Update yaml files for ArcGIS Insights 2022.1

### PR DESCRIPTION
**Pull request by fran8806**

### CODE REVIEW

**Comment by fran8806** [comment]
@chak7087 @vcove This PR includes all changes for Insights 2022.1. Please review.

File: examples/MySQL/dialect.yml [hunk 1]
```diff
@@ -98,21 +158,42 @@ sql functions:
   - name: create timestamp from components
     sql: >
       CASE WHEN $1 > 12 OR $1 < 1 OR $2 > 31 OR $2 < 1 OR
-        $3 > 24 OR $3 < 1 OR $4 > 60 OR $4 < 0 OR $5 > 60 OR $5 < 0 THEN NULL ELSE
+        $3 > 24 OR $3 < 0 OR $4 > 60 OR $4 < 0 OR $5 > 60 OR $5 < 0 THEN NULL ELSE
         STR_TO_DATE(CONCAT($0,'-',$1,'-',$2,' ',$3,':',$4,':',$5),
               CASE WHEN length($0) > 2 THEN '%Y-%m-%d %H:%i:%s' ELSE '%y-%m-%d %H:%i:%s' END)
       END
+# Description: Creates a timestamp from string 2 possible string timestamp format masks 'YYYY-MM-DD hh24:mi:ss' and 'YY-MM-DD hh24:mi:ss'
```
Comment (vcove) [inline examples/MySQL/dialect.yml:165]: This description doesn't make sense to me. I think it needs separate sentences

File: examples/MySQL/dialect.yml [hunk 2]
```diff
@@ -98,21 +158,42 @@ sql functions:
...
+# $1 Month part of the date component
+# $2 Day part of the date component
+# $3 Hour part of the time component
+# $4 Minutes part of the time component
+# $5 Seconds part of the time component
+
 
   - name: create timestamp from text
     sql: STR_TO_DATE($0, '$1')
+# Description: Creates a time from string field using a specified fromat mask
```
Comment (vcove) [inline examples/MySQL/dialect.yml:177]: "Creates a time from a string field using a specified format mask"

File: examples/MySQL/dialect.yml [hunk 3]
```diff
@@ -122,21 +203,102 @@ sql functions:
       CASE WHEN $1 > 12 OR $1 < 1 OR $2 > 31 OR $2 < 1 THEN NULL ELSE
       Date(STR_TO_DATE(CONCAT($0,'-',$1,'-',$2),
               CASE WHEN length($0) > 2 THEN '%Y-%m-%d' ELSE '%y-%m-%d' END)) END
+# Description: Creates a date from string 2 possible string date format masks  'YYYY-MM-DD' and 'YY-MM-DD'
```
Comment (vcove) [inline examples/MySQL/dialect.yml:206]: Should this be `Creates a date from strings. Two possible string date format masks:  'YYYY-MM-DD' and 'YY-MM-DD'`   ?

**Comment by vcove** [comment]
@fran8806 I've added changes for some of the comments

File: examples/MySQL/dialect.yml (hunk 1 above)
Comment (fran8806) [reply examples/MySQL/dialect.yml:165]: @vcove Can you suggest the change? Thanks.

**Comment by fran8806** [comment]
@vcove I have updated PR. Please check again. Thanks.

**Review by vcove — APPROVED** [review]

**Pull request merged**